src/RIO/PrettyPrint.hs

```diff
@@ -53,6 +58,21 @@ class (HasLogFunc env, HasStylesUpdate env) => HasTerm env where
   useColorL :: Lens' env Bool
   termWidthL :: Lens' env Int
 
+lookupEnvVar :: Text -> SimpleApp -> Maybe Text
+lookupEnvVar name env = M.lookup name $ (getConst . envVarsL Const) env
```

Comment:
@snoyberg, is there way to avoid global function and reuse code inside multiple methods of a class instance scope?

Reply:
No, but as long as you don't export it from the module that's fine.